src/module/Ui/templates/layout/partials/right-sidebar.phtml

```diff
@@ -1,12 +1,15 @@
 <?php if (!empty($this->registry()->get('rnav'))): ?>
-<aside class="side-element side-context">
+<?php $isCourse = $this->registry()->get('rnav-is-course');?>
+<aside class="side-element side-context <?php if (!empty($isCourse) && $isCourse) : echo 'side-context-course'; endif; ?> ">
     <div id="sidebar-toggle" class="layout-toggle">
         <i class="fa fa-bars"></i>
         <i class="fa fa-times"></i>
     </div>
     <div class="side-element-inner">
-        <i class="fa fa-asterisk"></i>
+        <?php if (empty($isCourse) || !$isCourse): ?>
```

Comment:
empty [compares to false](http://www.php.net/manual/en/types.comparisons.php), so you can simplify this:

```php
if (!$isCourse):
```